integration-tests-support/kafka/pom.xml

```diff
@@ -52,6 +52,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit4-mock</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-util</artifactId>
```

Comment:
Do we need `camel-util`?

Reply:
I used `CollectionHelper` from `camel-util` and you are right, I can refactor the code to not use it, therefore I can remove the dependency

Reply:
Removed